js/apps/admin-ui/src/client-scopes/ClientScopesSection.tsx

```diff
@@ -180,7 +180,7 @@ export default function ClientScopesSection() {
           addError("deleteErrorClientScope", error);
         }
       } else {
-        addError(t("notAllowedToDeleteAllClientScopes"), "error");
+        addError("notAllowedToDeleteAllClientScopes", "error");
```

Comment:
This is wrong, there is a value being passed as `error` which is just a string literal that will be omitted as the label does not format an error.

Not sure why this was done, likely to quickly work around the function signature requirements. This should be refactored to a plain call to `addAlert()`, for example:

```ts
addAlert(t("notAllowedToDeleteAllClientScopes"), AlertVariant.danger);
```